### ISSUE TITLE
Non-Descriptive Alt Text on Latest City of Boston Newspage

### ISSUE THREAD

**Issue by david-iterators** [CityOfBoston/Iterators#128]
Screengrab/videos (insert file names here)
N/A

Link
https://d8-stg.boston.gov/news

Node # (if known)

Type of User:  Anonymous

Environment
(Device > OS > Browser)
JAWS, Voice Over, Talkback

Trusted Tester Test

Brief Description
Line 1254 - “Intro Image” ALT text is insufficient to convey meaning and doesn’t make sense in or out of context to a screen reader user.

Note:  additional ALT text will be reviewed later.

Steps to reproduce
1)	Open source code and navigate to line 1254. 
2)	Alternately, navigate by graphic by: 
3)	Pressing G until hearing ALT text, “Intro Image graphic.” 

Desired Result
ALT text describes and conveys the meaning of image. 

Actual Result
ALT text does not describe and convey the meaning of image.

**Comment by subaha-cob** [comment]
@andrebtlima and @david-iterators  I am not completely sure which of the image(s) we are talking about here. Can you be more specific please. Thanks!

**Comment by subaha-cob** [comment]
There is no "Intro image".

**Comment by david-iterators** [comment]
This was an error that was picked up with a screen reader check.  We will clarify and post a new bug with a list of several images that require updates to the alt-text.  Therefore, you can close this issue and we will repost a different one.

**Comment by subaha-cob** [comment]
Closing issue per David's comment above.
